3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/cli.rb

```diff
@@ -260,15 +260,26 @@ def self.load_resolver(resolver_file, summary)
       # Load configuration from JSONs
       Contract Maybe[String] => ArrayOf[JsonFileHash]
       def self.load_targets(targets_path)
-        if targets_path.nil?
+        ids = []
```

Comment:
I am not sure how important that is here but you could instead do:
`ids = Hash.new(0)`
and at L273:
`ids[id] += 1`
and L278:
`duplicate_ids = ids.select { |_, c| c>1}.keys`
will save a few iterations

Reply:
Thanks, that's lack of Ruby experience, however having that `ids` is usually array of single element - I don't think we'll save enough many CPU cycles.